definitions/v7/gktorrent.yml

```
@@ -8,10 +8,11 @@ encoding: UTF-8
 followredirect: true
 # to fetch current domain use https://www.protege-torrent.com/Gktorrent
 links:
-  - https://www.gktorrents.cc/
+  - https://www.gktorrents.ac/
```

The new domain is incorrect, it should be
```suggestion
  - https://www.gktorrent.ac/
```
But this is academic since when the team import the latest indexer updates from Jackett this PR will be redundant.